ts/features/bonus/siciliaVola/screens/voucherGeneration/DisableAdditionalInfoScreen.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import { useRef } from "react";
+import { connect } from "react-redux";
+import { Dispatch } from "redux";
+import { SafeAreaView } from "react-native";
+import { NavigationEvents } from "react-navigation";
+import BaseScreenComponent from "../../../../../components/screens/BaseScreenComponent";
+import { emptyContextualHelp } from "../../../../../utils/emptyContextualHelp";
+import { setAccessibilityFocus } from "../../../../../utils/accessibility";
+import { IOStyles } from "../../../../../components/core/variables/IOStyles";
+import { H1 } from "../../../../../components/core/typography/H1";
+import { GlobalState } from "../../../../../store/reducers/types";
+import {
+  svGenerateVoucherBack,
+  svGenerateVoucherCancel
+} from "../../store/actions/voucherGeneration";
+import FooterWithButtons from "../../../../../components/ui/FooterWithButtons";
+import { navigateToSvSelectFlightsDateScreen } from "../../navigation/actions";
+import I18n from "../../../../../i18n";
+
+type Props = ReturnType<typeof mapDispatchToProps> &
+  ReturnType<typeof mapStateToProps>;
+
+const DisableAdditionalInfoScreen = (props: Props): React.ReactElement => {
+  const elementRef = useRef(null);
+  const backButtonProps = {
+    primary: false,
+    bordered: true,
+    onPress: props.back,
+    title: "Back"
+  };
+  const continueButtonProps = {
+    primary: false,
+    bordered: true,
+    onPress: props.navigateToSelectFlightsDateScreen,
+    title: "Continue"
+  };
+  return (
+    <BaseScreenComponent goBack={true} contextualHelp={emptyContextualHelp}>
+      <NavigationEvents onDidFocus={() => setAccessibilityFocus(elementRef)} />
+      <SafeAreaView
+        style={IOStyles.flex}
+        testID={"DisableAdditionalInfoScreen"}
```

Comment:
```suggestion
        testID={"DisabledAdditionalInfoScreen"}
```

Reply:
Fixed in [d99b38f](https://github.com/pagopa/io-app/pull/3212/commits/d99b38f901efc9658c26b2fb939fed3eeff739d8)